analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataFactory.cs

```diff
@@ -68,65 +66,37 @@ public static partial class NuGetMetadataFactory
 
         public static IEnumerable<MetadataReference> CreateNetStandard21()
         {
-            var packageDir = Path.Combine(PackagesFolder, @"NETStandard.Library.Ref.2.1.0\ref\netstandard2.1");
-            if (Directory.Exists(packageDir))
-            {
-                LogMessage($"Package found at {packageDir}");
-            }
-            else
-            {
-                LogMessage($"Package not found at {packageDir}");
-                PackageManager.InstallPackage("NETStandard.Library.Ref", SemanticVersion.ParseOptionalVersion("2.1.0"), ignoreDependencies: true, allowPrereleaseVersions: false);
-                if (!Directory.Exists(packageDir))
-                {
-                    throw new ApplicationException($"Test setup error: folder for downloaded package does not exist. Folder: {packageDir}");
-                }
-            }
+            var package = new Package("NETStandard.Library.Ref", "2.1.0", "netstandard2.1");
```

Comment:
This method was here because the command-line tool could install this thing and we had to use a different approach. Now it looks that we can remove this method and use `.Create(...)` on the caller side instead.